versions.props

```diff
@@ -1,24 +1,24 @@
 com.fasterxml.jackson.*:* = 2.13.3
-com.fasterxml.jackson.core:jackson-databind = 2.13.2.1
-com.github.ben-manes.caffeine:caffeine = 3.1.1
+com.fasterxml.jackson.core:* = 2.13.3
+com.github.ben-manes.caffeine:* = 3.1.1
 com.google.auto:* = 1.2.1
 com.google.auto.service:* = 1.0.1
-com.google.code.findbugs:jsr305 = 3.0.2
-com.google.guava:guava = 31.1-jre
-com.google.testing.compile:compile-testing = 0.19
+com.google.code.findbugs:* = 3.0.2
+com.google.guava:* = 31.1-jre
+com.google.testing.compile:* = 0.19
 com.palantir.goethe:* = 0.8.0
 com.palantir.jvm.diagnostics:* = 0.2.0
 com.palantir.safe-logging:* = 3.0.0
 com.palantir.tracing:* = 6.15.0
-com.squareup:javapoet = 1.13.0
-com.squareup.okhttp3:okhttp = 4.0.0
-com.uber.nullaway:nullaway = 0.10.1
-io.undertow:undertow-core = 2.2.19.Final
-net.bytebuddy:byte-buddy = 1.12.16
+com.squareup:* = 1.13.0
```

Comment:
This may be too broad, no?

Reply:
Yeah, this could pick up/conflict with a few other modules if we added dependencies on the them. Right now javapoet is the only `com.squareup` runtime dep, `okhttp3` and `okio` are test only deps

```
# runtime
com.squareup:javapoet:1.13.0 (2 constraints: 2b113eee)

[Test dependencies]
com.squareup.okhttp3:okhttp:4.0.0 (1 constraints: 0605ff35)
com.squareup.okio:okio:2.2.2 (1 constraints: 500c29fd)
```